api/api.yaml

```diff
@@ -167,6 +167,25 @@ paths:
         '500':
           $ref: '../result/request.yaml#/components/responses/InternalServerError'
 
+  /control/protocols:
+    post:
+      tags: ['Control']
+      operationId: set_protocol_state
+      description: >-
+        Sets all configured protocols to intended state
```

Comment:
started or stopped ?

Reply:
changed to mention `start` or `stop` state.